src/riot-rs-boards/dwm1001/Cargo.toml

```
@@ -10,6 +10,7 @@ edition = "2021"
 cortex-m-rt.workspace = true
 cortex-m.workspace = true
 
+riot-rs-debug = { workspace = true }
```

```suggestion
riot-rs-debug.workspace = true }
```

(for consistency, we might change all in one go)